### ISSUE TITLE
The light blue used for flat buttons and other clickable elements does not meet color contrast standards

### ISSUE THREAD

**Issue by nicklemmon** [SparkPost/matchbox#230]
![Screen Shot 2019-08-28 at 1 21 38 PM](https://user-images.githubusercontent.com/3613392/63878111-e7a2e300-c996-11e9-8356-292463dee27a.png)

Use the [WebAIM Color Contrast Checker](https://webaim.org/resources/contrastchecker/) and see that the ratio with white is about 3.5 to 1 when it will need to be 4.5 to 1 if this will be used for smaller text.

The color in question:
```
#2693C3
```

**Comment by jonambas** [comment]
Closing - addressed in hibana
